setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="django-soapbox",
-    version="1.6.1",
+    version="1.7.1",
```

Comment:
This should preferably be left for the author himself but if we are doing this change, then `1.7.0` is the next semantic release candidate.